app/assets/javascripts/AngularConfig.js.coffee

```diff
@@ -258,6 +258,8 @@ angular.module('dahlia.controllers',['ngSanitize', 'angular-carousel', 'ngFileUp
           squash: true
         alreadySubmittedId:
           squash: true
+        doubleSubmit:
```

Comment:
haha, these names are funny. seeing into the future...

alreadySubmit
doubleSubmit
tripleSubmit
foreverSubmit